### PR TITLE
feat(docs): Cookbook Onboard (AI Assistant) integration

### PR DESCRIPTION
**Pull request by ClockRide**
### What I Did

Extended `custom.js` with code to instantiate the Cookbook Onboard integration.

Resolves: #2197 

### How to Verify It
Launch the project in production or a dev environment and try out Cookbook Onboard by clicking the "Ask Cookbook" button in the bottom right corner of the docs :)

You can also use following preview url: https://apeworx-cookbook.vercel.app/ape/latest/userguides/quickstart.html

### Checklist

- [x] All changes are completed
- [ ] New test cases have been added
- [x] Documentation has been updated

### CODE REVIEW

**Comment by ClockRide** [comment]
I'm not sure if the "New test cases have been added" item on the checklist is applicable here, so I skipped it for now. Let me know if it is after all, and I'll be happy to make the necessary changes!

**Comment by antazoey** [comment]
note: you can use `docs:` PR title prefix instead of `feat(docs):` since this is not really a feature of the Ape tool itself

File: docs/_static/custom.js [hunk 1]
```diff
@@ -33,4 +33,24 @@ $(document).ready(function () {
     $("option[value='" + selected + "']").attr("selected", "selected");
     window.location = document.URL.replace(current, selected);
   });
+
+  // Cookbook Onboard (AI Assistant).
```
Comment (antazoey) [inline docs/_static/custom.js:37]: I don't see anywhere that controls for following text:

Screenshot 2024-08-01 at 16 38 42

`ApeWorX` does not have the correct casing, and where it is coming from (?) is mysterious...

File: docs/_static/custom.js [hunk 2]
```diff
@@ -33,4 +33,24 @@ $(document).ready(function () {
     $("option[value='" + selected + "']").attr("selected", "selected");
     window.location = document.URL.replace(current, selected);
   });
+
+  // Cookbook Onboard (AI Assistant). 
+  // API key is public so it's fine to just hardcode it here.
+  var COOKBOOK_PUBLIC_API_KEY = "eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJzdWIiOiI2NmFhY2QzYjZiYmI2MmQwOGY0ZjYzNzgiLCJpYXQiOjE3MjI0Njk2OTEsImV4cCI6MjAzODA0NTY5MX0.Xw4JO3X3T19NOGqPnsz_DYDEBdsxNYM9JkZK4k8ADs8";
```
Comment (antazoey) [inline docs/_static/custom.js:39]: What is the point of a public API key?

File: docs/_static/custom.js (hunk 1 above)
Comment (ClockRide) [reply docs/_static/custom.js:37]: Sorry for taking so long to reply! We have a CMS website where you can customize suggestions, view questions asked and the bot's responses, configure and update data sources, and much more! If you'd like an invite link, just reach out to us at hello@cookbook.dev from your corporate email, or please ask your colleague Chris (whom we chat with on Telegram) to add you to the chat :)

File: docs/_static/custom.js (hunk 2 above)
Comment (ClockRide) [reply docs/_static/custom.js:39]: It is used to make user interactions with Ask Cookbook available for your review in the CMS, allowing you to further tune replies by leaving feedback on conversations. Additionally, it enables you to dynamically configure the appearance of Ask Cookbook using the same CMS

**Comment by ClockRide** [comment]
bumping :)

**Comment by antazoey** [comment]
The file being changed no longer exists.
We are using https://github.com/ApeWorX/sphinx-ape for docs now

**Pull request closed without merging**